src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -2593,6 +2696,701 @@ interface IB
             );
         }
 
+        [Fact]
+        public void Inheritance_12()
+        {
+            var source =
+@"class A
+{
+    public object X { get; }
+    public object Y { get; }
+}
+data class B(object X, object Y) : A
+{
+    public object X { get; }
+    public object Y { get; }
+}";
+            var comp = CreateCompilation(source);
+            comp.VerifyDiagnostics(
```

Comment:
>VerifyDiagnostics [](start = 17, length = 17)

Per [https://github.com/dotnet/csharplang/pull/3527](https://github.com/dotnet/csharplang/pull/3527): 'Members are synthesized unless an accessible concrete (non-abstract) member with a "matching" signature is 
either inherited or declared in the record body.'

Since such a member is inherited, it should not be synthesized.